.drone/drone.yml

```diff
@@ -508,6 +508,52 @@ volumes:
   name: docker
 ---
 depends_on:
+- Publish development Linux alloy container
+image_pull_secrets:
+- dockerconfigjson
+kind: pipeline
+name: Deploy to deployment_tools
+platform:
+  arch: amd64
+  os: linux
+steps:
+- commands:
+  - apk update && apk add git
+  - echo "$(sh ./tools/image-tag-docker)" > .tag-only
+  - echo "grafana/alloy-dev:$(sh ./tools/image-tag-docker)" > .image-tag
+  image: alpine
+  name: Create .image-tag
+- image: us.gcr.io/kubernetes-dev/drone/plugins/updater
+  name: Update deployment_tools
+  settings:
+    config_json: |
+      {
+        "git_committer_name": "updater-for-ci[bot]",
+        "git_author_name": "updater-for-ci[bot]",
+        "git_committer_email": "119986603+updater-for-ci[bot]@users.noreply.github.com",
```

Comment:
any danger in exposing this?

Reply:
I doubt it, you still need the private key to be able to do anything and other teams have been using the same workflow with the same settings AFAIK (including grafana/agent).